.gitignore

```diff
@@ -25,3 +25,5 @@ yarn-error.log*
 storybook-static
 
 /__generated__/
+
+.vscode/
```

Comment:
I think it's good to keep the common settings shared among team members.
As far as I know, it's still possible that one can customize his/her settings using `user settings`.

Reply:
I decided to install a vscode extension that prrovides unique theme for each workspace. I didn't want to enforce this so I put vscode settings to gitognore. You have a fair point through. What do you think?

Reply:
https://marketplace.visualstudio.com/items?itemName=stuart.unique-window-colors

Reply:
Problem solved with a global gitignore file.